src/docbkx/chapter12-getting-started-desktop.xml

```diff
@@ -270,23 +270,44 @@
 			</imageobject>
 		</mediaobject>
 		<section>
-		<title>Filter requirement</title>
-		<para>To wire a filter you need to set up a dependency on either of
-			it's outcomes. All components have a button for selecting filter
-			outcomes in the top-right corners of their configuration dialogs.
-			Click this button to select a filter outcome to depend on. If you
-			have
-			multiple filters you can chain these simply by having dependent
-			outcomes of the individual filters.
-		</para>
-		</section>
-		<tip>
+			<title>Filter requirement</title>
+			<para>To wire a filter you need to set up a dependency on either of
+				it's outcomes. All components have a button for selecting filter
+				outcomes in the top-right corners of their configuration dialogs.
+				Click this button to select a filter outcome to depend on.
+			</para>
+
+			<para>If you have multiple filters you can chain these simply by having dependent
+				outcomes of the individual filters. This will require all outcomes in the chain
+				to be valid for a record to be passed to the component (AND logic).
+			</para>
+
+			<mediaobject>
+				<imageobject>
+					<imagedata fileref="filter_chain.png"
+							   format="PNG" scalefit="1" />
+				</imageobject>
+				<caption>
+					<para>Chained filters</para>
+				</caption>
+			</mediaobject>
+
 			<para>
-				In the bottom part of the canvas, a help message is displayed
-				instructing what needs to be done in current moment to build a valid
-				job.
+				Using "Link to...", it is also possible to wire several filters to a component
+				in a kind of diamond shape. In that case, if any of the the filters are valid,
```

Comment:
"valid" maybe not so precise.

What about "... if any of the filter requirements are met,"